tags/fr.json

```diff
@@ -48,7 +48,7 @@
     "tag:amenity=bicycle_parking capacity<=60 capacity>20": "Parking vélo (capacité 20> <60)",
     "tag:amenity=bicycle_parking capacity>60": "Parking vélo (capacité +60)",
     "tag:amenity=bicycle_rental": "Location de vélos",
-    "tag:amenity=biergarten": "",
+    "tag:amenity=biergarten": "Biegarten",
```

Comment:
Are you sure, it's "Biegarten"?

Reply:
I can translate it literally by "Jardin de bière" but it is not explicit because it does not exist in France.
Maybe like in Wikipedia Fr (https://fr.wikipedia.org/wiki/Biergarten) by "**Brasserie en plein air**"